movie-browser/src/components/DisplayMovies/DisplayMovies.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
why do you need 2 types of `Movies` component? The name is also ambiguous. I would recommend `<MovieList />` or something like that

Reply:
But what's the idea behind these 2 components? What will they contain (each of them) ?

Reply:
Going to restructure to rename to MovieList and get rid of FeatureMovie and TrendingMovie components. In MovieList will make the call to the API and store the the data in state